### DISCUSSION TITLE
Dependency Parser

### DISCUSSION THREAD

**Discussion by mustafa0x** [CAMeL-Lab/camel_tools#125] [Ideas]
I noticed [farasa.qcri.org](https://farasa.qcri.org) has a dependency parser (although results seem iffy). Any plans to add such a tool to camel?

Thanks for this effort!

**Comment by owo** [comment]
Yes! Stay tuned for more info on that :)

**Comment by mustafa0x** [comment]
I noticed stanza has a dependency parser but results don't seem to make much sense. https://github.com/stanfordnlp/stanza, http://stanza.run
